process/dataValidators/txValidator.go

```diff
@@ -76,14 +77,27 @@ func (txv *txValidator) CheckTxValidity(interceptedTx process.InterceptedTransac
 		return nil
 	}
 
+	// for relayed v3, we allow sender accounts that do not exist
+	isRelayedV3 := common.IsRelayedTxV3(interceptedTx.Transaction())
+	hasValue := hasTxValue(interceptedTx)
+	shouldAllowMissingSenderAccount := isRelayedV3 && !hasValue
 	accountHandler, err := txv.getSenderAccount(interceptedTx)
-	if err != nil {
+	if err != nil && !shouldAllowMissingSenderAccount {
 		return err
 	}
 
 	return txv.checkAccount(interceptedTx, accountHandler)
 }
 
+func hasTxValue(interceptedTx process.InterceptedTransactionHandler) bool {
+	txValue := interceptedTx.Transaction().GetValue()
+	if check.IfNilReflect(txValue) {
+		return false
+	}
+
+	return big.NewInt(0).Cmp(txValue) < 0
```

Comment:
Maybe directly use `Sign()` method of `txValue`, without instantiating a new big int (zero)?

Reply:
pushed